.github/workflows/stable-branch-sync.yml

```diff
@@ -0,0 +1,43 @@
+name: Stable Branch Sync
+
+permissions:
```

Comment:
If we use a GITHUB_TOKEN for making this commit and PR, then CI checks will fail to run. GitHub Action workflows as a general rule cannot be triggered by other actions made with the GITHUB_TOKEN.

For this reason, we use a personal access token for making commits and PRs in our workflows elsewhere. I think we'll need to use a token here as well.